README.md

```diff
@@ -181,14 +181,13 @@ This example uses [fd](https://github.com/sharkdp/fd).
 Good ol' unix philosophy to the rescue.
 
 ```sh
-sd 'from "react"' 'from "preact"' $(fd --type file)
+fd --type file --exec sd 'from "react"' 'from "preact"'
 ```
 
 Same, but with backups (consider version control).
 
 ```bash
-for file in $(fd --type file); do
-  cp "$file" "$file.bk"
-  sd 'from "react"' 'from "preact"' "$file"; 
-done
+fd --type file --exec \
+  cp {} {}.bk; \
+  sd 'from "react"' 'from "preact"'
```

Comment:
```suggestion
fd --type file --exec cp {} {}.bk \; --exec sd 'from "react"' 'from "preact"'
```

Had to change this example since the previous version gets parsed as two separate commands

```bash
fd --type file --exec cp {} {}.bk
```

and

```bash
sd 'from "react"' 'from "preact"'
```

Where `sd` is stuck trying to read from `stdin` since it doesn't get passed a file list